CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ _All notable changes to this project will be documented in this file. This proje
 
 <!-- INSERT HERE -->
 
+<!-- NEXT VERSION
```

Comment:
Don't use title/comments. It confuses the auto-generation script

Reply:
Yeah, that's just temporary until we run the auto-generation script.